Solutions/FlareSystemsFirework/Workbooks/FlareSystemsFireworkOverview.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
One more addition regarding the workbook, please add preview images of the workbook in a new sub-folder, `/Solutions/FlareSystemsFirework/Workbooks/Images/Preview`. 
Preview images should cover the full workbook, with each image having 2 versions, one using Light Mode and the other using Dark Mode in the Azure Portal.

See https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Cloudflare/Workbooks/Images/Preview for example.

Reply:
Done!